samples/aml_monitoring/requirements.txt

```diff
@@ -1,2 +1,3 @@
 azure-functions
+azure-functions-durable>=1.0.0b5
```

Comment:
I assume the tooling will usually try to grab the latest, but maybe for consistency we should update this to 1.0.0b6?

Reply:
ah yes, I missed this. Will fix! 🦑

Reply:
Done!